### DISCUSSION TITLE
Error when importing API spec

### DISCUSSION THREAD

**Discussion by ethan-cassel-mace** [GitbookIO/community#739] [4. Open API]
I'm getting an error when importing my api spec from url here:

https://service-run-api.muir.ai/carbon_origin/openapi.json

here's a screen capture of the exaperiance:

https://github.com/user-attachments/assets/d3e1192c-5885-4c74-b51e-a4e650cbc766

**Comment by aikoii** [comment]
👋 @ethan-cassel-mace, it looks like there is a CORS policy error when trying to fetch data from `https://service-run-api.muir.ai`. The server isn’t including the required `Access-Control-Allow-Origin` header in its responses, which is necessary for cross-origin requests from `https://app.gitbook.com`. Reconfiguring this should resolve the issue.

**Comment by ethan-cassel-mace** [comment]
ah got it, that makes sense thanks so much for the reply!
